.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -70,7 +70,7 @@ body:
     id: privileged_rootless
     attributes:
       label: Privileged Or Rootless
-      description: Are you running the containers as privileged or non-root user?
+      description: Are you running the containers as privileged or non-root user?  Note that using `su` does not establish a proper login session required for running Podman as a non-root user.
```

Comment:
good idea, you can also link directly to https://github.com/containers/podman/blob/main/troubleshooting.md#solution-28 to show some alternatives

Reply:
I like that!  PTanotherL